<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1522</width>
    <height>718</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="GKChess::UI::BoardView" name="boardView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1522</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBoard">
    <property name="title">
     <string>&amp;Board</string>
    </property>
    <widget class="QMenu" name="menu_Load_Position">
     <property name="title">
      <string>&amp;Load Position</string>
     </property>
     <addaction name="actionLoad_FEN_in_Clipboard"/>
     <addaction name="separator"/>
     <addaction name="actionLoad_PGN_in_Clipboard"/>
     <addaction name="actionLoadPGN_File"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <addaction name="actionPosition_to_Clipboard"/>
    </widget>
    <widget class="QMenu" name="menu_New_Position">
     <property name="title">
      <string>&amp;New Position</string>
     </property>
     <addaction name="actionStandard_Starting_Position"/>
     <addaction name="actionChess960_Starting_Position"/>
    </widget>
    <addaction name="menu_New_Position"/>
    <addaction name="menu_Load_Position"/>
    <addaction name="action_Recent_Positions"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionRandom_Chess960_Position"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>&amp;Widgets</string>
    </property>
    <addaction name="actionMove_History"/>
    <addaction name="action_Engine_Control"/>
    <addaction name="actionOpening_Book_Reader"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuBoard"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_FEN_in_Clipboard">
   <property name="text">
    <string>&amp;FEN String in Clipboard</string>
   </property>
   <property name="whatsThis">
    <string>Populates the board from a FEN string in your clipboard</string>
   </property>
  </action>
  <action name="actionLoad_PGN_in_Clipboard">
   <property name="text">
    <string>&amp;PGN String in Clipboard</string>
   </property>
   <property name="whatsThis">
    <string>Populates the board from a PGN string in your clipboard</string>
   </property>
  </action>
  <action name="actionLoadPGN_File">
   <property name="text">
    <string>PGN File</string>
   </property>
   <property name="whatsThis">
    <string>Populates the board from a PGN string in a file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="whatsThis">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionRandom_Chess960_Position">
   <property name="text">
    <string>&amp;Chess960 Position Generator</string>
   </property>
   <property name="whatsThis">
    <string>Generate Chess960 positions at random or choose specific ones</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="whatsThis">
    <string>About this application</string>
   </property>
  </action>
  <action name="actionOpening_Book_Reader">
   <property name="text">
    <string>&amp;Book Reader</string>
   </property>
   <property name="whatsThis">
    <string>Read an opening book</string>
   </property>
  </action>
  <action name="actionStandard_Starting_Position">
   <property name="text">
    <string>&amp;Standard</string>
   </property>
   <property name="whatsThis">
    <string>The standard chess starting position</string>
   </property>
  </action>
  <action name="actionChess960_Starting_Position">
   <property name="text">
    <string>&amp;Chess960</string>
   </property>
   <property name="whatsThis">
    <string>A random Chess960 position</string>
   </property>
  </action>
  <action name="actionMove_History">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move &amp;History</string>
   </property>
   <property name="whatsThis">
    <string>Shows the PGN move history</string>
   </property>
  </action>
  <action name="actionPosition_to_Clipboard">
   <property name="text">
    <string>Position to &amp;Clipboard (FEN)</string>
   </property>
   <property name="whatsThis">
    <string>Exports the position to the clipboard</string>
   </property>
  </action>
  <action name="action_Engine_Control">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Engine Control</string>
   </property>
   <property name="whatsThis">
    <string>Analyse the position using a UCI compatible engine</string>
   </property>
  </action>
  <action name="action_Recent_Positions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Recent Positions</string>
   </property>
  </action>
  <action name="action_Whats_this">
   <property name="text">
    <string>&amp;What's this?</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GKChess::UI::BoardView</class>
   <extends>QWidget</extends>
   <header>gkchess_boardview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
